### PR TITLE
Add resource quota infomation

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Added a doc describing ResourceQuotas and LimitRanges to Kyma/Details

**Related issue(s)**
#4872

### CODE REVIEW

File: docs/kyma/03-05-resource-quotas.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Resource quotas
+type: Details
+---
+
+[Resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are a convenient way to manage the consumption of resources in a Kyma cluster. You can easily set resource quotas for every Namespace you create through the Console UI.
+
+When you click **Create Namespace**, you can define:
+  - Total Memory Quotas, which limit the overall memory consumption by the Namespace by creating a ResourceQuota object.
```
Comment (klaudiagrz) [inline docs/kyma/03-05-resource-quotas.md]:
```suggestion
  - Total Memory Quotas, which limit the overall Namespace memory consumption by creating ResourceQuota objects.
```

Comment (klaudiagrz) [reply docs/kyma/03-05-resource-quotas.md]: Resource quotas or ResourceQuotas? Whichever you choose, be consistent.

File: docs/kyma/03-05-resource-quotas.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - Limits per Container, which limit the memory consumption for individual Containers in the Namespace by creating a [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) object.
```
Comment (klaudiagrz) [inline docs/kyma/03-05-resource-quotas.md]:
```suggestion
  - Limits per container, which limit the memory consumption for individual containers in the Namespace by creating [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) objects.
```

Comment (klaudiagrz) [reply docs/kyma/03-05-resource-quotas.md]: As far as I remember, we do not capitalize containers.

File: docs/kyma/03-05-resource-quotas.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+To manage existing resource quotas in a Namespace, select that Namespace in the **Namespaces** view of the Console and go to the **Resources** tab. This view allows you to edit or delete the existing limits.
```
Comment (klaudiagrz) [inline docs/kyma/03-05-resource-quotas.md]: resource quotas / ResourceQuotas ?

File: docs/kyma/03-05-resource-quotas.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+>**TIP:** If you want to manage ResourceQuotas and LimitRanges directly from the terminal, follow [this](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/) comprehensive Kubernetes guide.  
+
+git config --global user.email "tomasz.papiernik@sap.com" ; git config --global user.name "tomekpapiernik"
```
Comment (piotrmsc) [inline docs/kyma/03-05-resource-quotas.md]: 🎉

File: docs/kyma/03-05-resource-quotas.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+[Resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are a convenient way to manage the consumption of resources in a Kyma cluster. You can easily set resource quotas for every Namespace you create through the Console UI.
+
+When you click **Create Namespace**, you can define:
+  - Total Memory Quotas, which limit the overall memory consumption by the Namespace by creating a ResourceQuota object.
+  - Limits per Container, which limit the memory consumption for individual Containers in the Namespace by creating a [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) object.
+
+To manage existing resource quotas in a Namespace, select that Namespace in the **Namespaces** view of the Console and go to the **Resources** tab. This view allows you to edit or delete the existing limits.
+
+>**TIP:** If you want to manage ResourceQuotas and LimitRanges directly from the terminal, follow [this](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/) comprehensive Kubernetes guide.
```
Comment (klaudiagrz) [inline docs/kyma/03-05-resource-quotas.md]: resource quotas / ResourceQuotas ?

File: docs/kyma/03-05-resource-quotas.md (hunk 4 above)
Comment (klaudiagrz) [inline docs/kyma/03-05-resource-quotas.md]: What is that?

File: docs/kyma/03-05-resource-quotas.md (hunk 1 above)
Comment (tomekpapiernik) [reply docs/kyma/03-05-resource-quotas.md]: Resource quotas are set through a ResourceQuota Kubernetes object.

File: docs/kyma/03-05-resource-quotas.md (hunk 3 above)
Comment (tomekpapiernik) [reply docs/kyma/03-05-resource-quotas.md]: Resource quotas are set through a ResourceQuota Kubernetes object.

File: docs/kyma/03-05-resource-quotas.md (hunk 2 above)
Comment (tomekpapiernik) [reply docs/kyma/03-05-resource-quotas.md]: It's weird to capitalize Pods but not containers, but okay ;p

File: docs/kyma/03-05-resource-quotas.md (hunk 1 above)
Comment (tomekpapiernik) [reply docs/kyma/03-05-resource-quotas.md]: Oh, and there's a single ResourceQuota object which limits how much resources the Namespace can consume.

**Pull request merged**